modules/340-monitoring-deckhouse/.helmignore

```diff
@@ -0,0 +1,7 @@
+crds
```

Comment:
There are more entries in the helmignore file than needed.

Reply:
Cleaned